test/helpers/api-integration/api-classes.js

```diff
@@ -59,6 +59,31 @@ export class ApiGroup extends ApiObject {
 
     this._docType = 'groups';
   }
+
+  async addChat (chat) {
+    let group = this;
+
+    if (!chat) {
+      chat = {
+        id: 'Test_ID',
+        text: 'Test message',
+        flagCount: 0,
+        timestamp: Date(),
+        likes: {},
+        flags: {},
+        uuid: group.leader,
+        contributor: {},
+        backer: {},
+        user: group.leader,
+      };
+    }
+
+    let update = { chat };
+
+    this.update(update, '$push');
+
+    return this;
```

Comment:
this is wrong I think, this.update is an async function so you have to call `await this.update` and since `this.update` already returns `this` you may just do `return await this.update`